src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -33,6 +33,17 @@ Git: 0.7-summer-2025-1037-g9bd1d5052
     <item name="JPEG" value="0x00"/>
   </enum>
 
```

Comment:
The StreamUsageEnum is used by multiple clusters. Could this reside here or needs to be put in the global-enums.xml?

Reply:
It's not a true global, it's scoped to a discrete set of camera clusters.  Given how the spec has defined it (within the cluster), Alchemy creates XML that lists all using clusters.  Code gen deals with this correctly.